StoreKitUnitTests/TransactionsManagertSK1Tests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  TransactionsManagertSK1Tests.swift
+//
+//  Created by Juanpe Catalán on 10/12/21.
+
+import Nimble
+@testable import RevenueCat
+import XCTest
+
+class TransactionsManagertSK1Tests: StoreKitConfigTestCase {
+
+    var mockReceiptParser: MockReceiptParser!
+    var transactionsManager: TransactionsManager!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        mockReceiptParser = MockReceiptParser()
+
+        transactionsManager = TransactionsManager(receiptParser: mockReceiptParser)
+    }
+
+    func testSK1CheckCustomerHasTransactionsParserIsCalled() {
+        _ = transactionsManager.sk1CheckCustomerHasTransactions(receiptData: Data())
+        expect(self.mockReceiptParser.invokedReceiptHasTransactions).to(beTrue())
+    }
+
+    func testSK1CheckCustomerHasTransactionsCalculatedFromReceiptData() {
+        mockReceiptParser.stubbedReceiptHasTransactionsResult = true
+        let receivedHasTransactionsValue = transactionsManager.sk1CheckCustomerHasTransactions(receiptData: Data())
+        expect(receivedHasTransactionsValue).to(beTrue())
```

Comment:
you can simplify the syntax of these to 
```swift
expect(receivedHasTransactionsValue) == true
```

Although it's just a matter of preference, I'm good with either

Reply:
much better and readable :)